apps/dotcom/client/src/tla/app/TldrawApp.ts

```diff
@@ -271,6 +282,12 @@ export class TldrawApp {
 		return { newFile, editorStoreSnapshot }
 	}
 
+	isFileOwner(fileId: TldrawAppFileId) {
```

Comment:
i assume eventually, non file owners should be able to publish, eh? but for now this is fine

Reply:
yes, definitely still useful — sorry, this code is good. i meant this as a comment more like we'll need an additional permissions check later for publishing by multiple editors later.

Reply:
Yeah. 

With @steveruizok we discussed different permissions levels. Publishing / deleting a published document is meant for `Full access` level. Right now that's only the owner, but when we will have groups then the whole group will have `Full access`.